app/bundles/PageBundle/EventListener/SearchSubscriber.php

```diff
@@ -89,7 +89,7 @@ public function onGlobalSearch(MauticEvents\GlobalSearchEvent $event)
                 foreach ($pages as $page) {
                     $pageResults[] = $this->templating->getTemplating()->renderResponse(
                         'MauticPageBundle:SubscribedEvents\Search:global.html.php',
-                        ['page' => $page]
+                        ['page' => is_array($page) ? $page[array_key_first($page)] : $page]
```

Comment:
I had to check here what the data structure is. I can see it is like 
```php
[[Page], [Page]]
```
So we'd have to iterate through the data structure like we did and then get the page entity with `$page[0]`. This is not the data structure it should return. I dug deeper and found that this is the query being searched for:

```sql
SELECT p, (SELECT count(fs.id) 
FROM MauticFormBundle:Submission fs 
WHERE fs.page = p) as submission_count 
FROM Mautic\PageBundle\Entity\Page p 
LEFT JOIN p.category c 
WHERE p.title LIKE :par0 OR p.alias LIKE :par0 
ORDER BY p.title ASC
```
The form submission shouldn't be there. Can you please check where it's coming from? The query object probably does not get cleared somewhere. Fixing that query is the right fix for this bug.

Reply:
I absolutely agree with you that fixing the query will be better. I just thought the `submission_count` field was necessary. I'll figure it out and try to fix this query.

Reply:
It came from the [PR 9693](https://github.com/mautic/mautic/pull/9693)

Reply:
@escopecz could you please look at a new solution.

I added a new `submissions' field to the Page entity. As a result, the array looks like [[Page], [Page]], and each `Page` includes `submissions`.

I left the existing query and declared `submission_count` as hidden. This way it can be used in the order by clause without being in the result.

What do you think?